sky/utils/kubernetes/kubernetes_deploy_utils.py

```diff
@@ -26,6 +26,48 @@
 DEFAULT_KUBECONFIG_PATH = os.path.expanduser('~/.kube/config')
 
 
+def check_ssh_cluster_dependencies(
+        raise_error: bool = True) -> Optional[List[str]]:
```

Comment:
unused arg?

Reply:
Yes, just keep the signature same with `check_port_forward_mode_dependencies`, I think we can merge these two functions later